README.md

```diff
@@ -2,198 +2,78 @@
- (If this is the first time you create the databases, you will see some warnings which are pointless):
-
-  ```shell
-  $> ./scripts/recreate_db.sh
-```
-7. We will populate data:
-
-###### Clean environment (recommanded)
-
-- With django migrations
-```shell
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py migrate --settings=backend.settings.dev
-```
-
-###### With fixtures (fixtures container has been run from hlf-k8s, old behavior for testing)
-
-data in fixtures are relative to the data already set in the ledger if the fixtures container instance succeeded
-
-Two solutions:
-- With django migrations + load data
-```shell
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py migrate --settings=backend.settings.dev
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 python backend/manage.py loaddata ./fixtures/data_owkin.json --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 python backend/manage.py loaddata ./fixtures/data_chu-nantes.json --settings=backend.settings.dev
-```
-- From dumps:
-```shell
-  $> ./scripts/populate_db.sh
-```
-If you don't want to replicate the data in the ledger, simply run the django migrations.
-
-Populate media files
-```shell
-  $> ./scripts/load_fixtures.sh
-```
-It will clean the `medias` folders and create the `owkin` and `chu-nantes` folders in the `medias` folder.
-
-
-8. Optional: Create a superuser in your databases:
-```shell
-BACKEND_ORG=owkin BACKEND_DEFAULT_PORT=8000 ./backend/manage.py createsuperuser --settings=backend.settings.dev
-BACKEND_ORG=chu-nantes BACKEND_DEFAULT_PORT=8001 ./backend/manage.py createsuperuser --settings=backend.settings.dev
-```
-
-## Getting started 2: Linking the app with Hyperledger Fabric
-
-
-### Make the hlf-k8s available to the app
-
-[See here](https://github.com/SubstraFoundation/hlf-k8s#network).
+## Running a development instance
 
-### Install rabbitmq
+For the local installation of substra-backend (and companions), please refer to the [setup documentation](https://doc.substra.ai/setup/local_install_skaffold.html).
 
-```shell
-sudo apt-get install rabbitmq-server
+With [hlf-k8s](https://github.com/SubstraFoundation/hlf-k8s) already running and requirements fulfilled, this should boils down to:
+```sh
+skaffold dev
 ```
 
-### Launch celery workers/scheduler and celery beat
+This will spawn several pods in two different namespaces to simulate several organizations: 'org-1' and 'org-2'.
+Each organization will have:
+- a postgres database
+- a rabbitmq message broker
+- a docker registry
+- a kaniko cache warmer
+- a [celery beat](./charts/substra-backend/templates/deployment-celerybeat.yaml)
```

Comment:
"a" bit disturbing the list with <a> word. I would suggest having simply the name of the different components :-) 
```suggestion
- Postgres database
- RabbitMQ message broker
- Docker registry
- Kaniko cache warmer
- [Celery beat](./charts/substra-backend/templates/deployment-celerybeat.yaml)
```